src/services/company.js

```diff
@@ -0,0 +1,16 @@
+const Offer = require("../models/Company");
+
+class CompanyService {
+    // TODO: Use typedi or similar
+    constructor() {
+
+    }
+
+    async getOffers(companyId) {
+        const offers = await Offer.find({ owner: companyId });
+
+        return offers;
+    }
+}
```

Comment:
```suggestion
    getCurrentOffers(companyId) {
        return Offer.find({ 
            owner: companyId, 
            publishEndDate: { $gte: { Date.now() } }
        });
    }
}
```
I would rename this to `getCurrentOffers`, with the required changes to the filter, in order to avoid problems when checking the amount of offers for the limit. We can still keep a `getOffers` which I believe will be useful in the future, but for now, this needs to be changed.